### PR TITLE
Update whats-new-in-preview.md

### PR DESCRIPTION
**Pull request by msbemba**
removed

":And we've got one-whole setting to announce: **Modern authentication**. Yes, there is a new setting to turn on **Modern authentication**! To check it out, go to **Settings** > **Services & add-ins** > **Modern authentication**." As this feature isn't added to the settings yet.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by cmcatee-MSFT** [comment]
@adefowler Please review changes. Thanks.

File: Office365-Admin/whats-new-in-preview.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ You might see a new recommendation pop-up in your admin center - we just added 5
 - **Get monthly Office feature updates** - We've gotten feedback from our very small customers that when they get monthly Office feature updates, their users are  happier. So, if you're a very small business and you're currently get your Office feature updates every six months, you'll see this recommendation.
 
 **Settings** <br>
-As for settings, there have been quite a few changes. Mostly, just updating the existing settings to the new admin center style. As we're moving forward and add new settings that you've never seen before, we'll start mentioning them here. And we've got one-whole setting to announce: **Modern authentication**. Yes, there is a new setting to turn on **Modern authentication**! To check it out, go to **Settings** > **Services & add-ins** > **Modern authentication**.
+As for settings, there have been quite a few changes. Mostly, just updating the existing settings to the new admin center style. As we're moving forward and add new settings that you've never seen before, we'll start mentioning them here.
```
Comment (nenonix) [inline Office365-Admin/whats-new-in-preview.md]:
```suggestion
As for settings, there have been quite a few changes. Mostly, just updating the existing settings to the new admin center style. As we move forward and add new settings that you've never seen before, we'll start mentioning them here. 
```

**Comment by mypil** [comment]
@adefowler - We would like to follow up. Can you please assist to check, approve and merge the content of this PR based on issue #549?

Thank you.

**Comment by mypil** [comment]
@adefowler - We would like to follow up on this. Can you please assist to check, approve and merge the content of this PR based on issue #549?

Thank you.

cc: @AndreaBarr @scotvidican

**Comment by adefowler** [comment]
Not all features are available to everyone as they start to roll out....the feature might be in "Targeted release" or only available to a certain SKU.

**Pull request closed without merging**